integrationtests/tests/fixtures/mod.rs

```diff
@@ -973,22 +974,26 @@ impl FederationTest {
         }
     }
 
-    /// Has every federation node send new consensus proposals then process the outcome.
-    /// If the epoch has empty proposals (no new information) then panic
+    /// Runs `n` epochs in the federation (each guardian node)
+    ///
+    /// Call in tests when new epoch conditions are already
+    /// in place, to trigger new epoch generation.
+    ///
+    /// Wallet chain height can trigger epoch randomly, but since
+    /// they will be processed along-side any other epochs you
+    /// already expect, nothing is being done about it. It should
+    /// not matter, but is worth pointing out.
+    ///
+    /// # Panics
+    ///
+    /// Panics if there's an empty proposal.
     #[allow(clippy::await_holding_refcell_ref)]
```

Comment:
We don't need this anymore, right?

Reply:
https://github.com/fedimint/fedimint/pull/1599 fixes it